atmat/lattice/atsimplering.m

```diff
@@ -0,0 +1,124 @@
+function ring = atsimplering(energy,circumference,hnum,...
+    qx,qy,vrf,alpha,varargin)
+%ATSIMPLERING   Creates a "simple ring"
+%
+% A "simple ring" consists of:
+% - A RF cavity
+% - a 6x6 linear transfer map
+% - a detuning and chromaticity element
+% - a simplified quantum diffusion element which provides the equilibrium
+% emittance, radiation damping and energy loss
+%
+%RING=ATSIMPLERING(ENERGY,CIRCUMFERENCE,HARMONIC_NUMBER,HTUNE,VTUNE,...
+%                  VRF,ALPHAC)
+% ENERGY:           nominal energy [eV]
+% CIRCUMFERENCE:    ring circumference [m]
+% HARMONIC_NUMBER:  harmonic number
+% HTUNE:            horizontal tune
+% VTUNE:            vertical tune
+% VRF:              RF voltage [V]
+% ALPHAC:           momentum compaction factor
+%
+% RING:             generated simple lattice
+%
+%RING=ATSIMPLERING(...,'name',name,...)
+%       specify the name of the generated lattice. Default: ''
+%
+%RING=ATSIMPLERING(...,'Particle',particle,...)
+%       specify the particle. Default: 'relativistic'
+%
+%RING=ATSIMPLERING(...,'Qpx',xsi_h,'Qpy',xsi_v,...)
+%       specify the chromaticites. Default: 0
+%
+%RING=ATSIMPLERING(...,'A1',A1,'A2',A2,'A3',A3,...)
+%       specify the amplitude detuning coefficients. Default: 0
+%
+%RING=ATSIMPLERING(...,'betax',betax,'betay',betay,...)
+%       specify the beta values at origin [m]. Default: 1
+%
+%RING=ATSIMPLERING(...,'betax',betax,'betay',betay,...)
+%       specify the alpha values at origin. Default: 0
+%
+%RING=ATSIMPLERING(...,'emitx',emitx,'emity',emity,...)
+%       specify the equilibrium emittance. Default: 0, meaning no quantum
+%       diffusion
+%
+%RING=ATSIMPLERING(...,'taux',taux,'tauy',tauy,'tauz',tauz,...)
+%       specify the damping times. Default: 0, meaning no damping
+%
+%RING=ATSIMPLERING(...,'U0',U0,...)
+%       specify the emergy loss per turn [eV]. Default: 0
```

Comment:
You mention here that the default for U0 is 0, but there is no getoption for it below. I don't see wher eit is passed to SimpleQuantDiff